demo/simple-column-sorting-pipeline.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
I admittedly, pivoted away from our original plans, but I feel this is the approach to take.

With regards to keeping the existing DataSourceSorter paradigm..

I think the point is not to get rid of Steve Wirt's code but decide who is owning what portion (us or the application developer). The code now clearly puts the onus on us to maintain all uses cases for sorting all Hypergrid users could potentially want.  The scaffold for sorting has opinions in that it assumes the user wants a straightforward multi-column sort for instance. What if they want only single column sort? What if they want to not sort certain columns against certain other columns? What if they want they only want to have certain cells in a column fixed and unsorted? I think having this scaffold to support simply comparators would not suffice and we need to emphasize that building your own sorting pipeline for in-memory is easy to do.

I do agree that DataSourceSorter has nice patterns, but I can't anticipate being generic enough. The idea is to empower the user and not have them dependent on our patterns.

With regards to simply having a default comparator option...
I don't think any defaults should be created. Maybe always alphanumeric sorting will be standard sorting, but what happens for different data structures (especially since our scaffold wants to be able to do multi-column sort) what do we assume the comparator is for a sparkline or a button even? This puts the maintenance of this code on us where as if we just make it easy for the user to make their own sorter then they can control it fully.